up42/glossary.py

```diff
@@ -10,16 +12,17 @@ class CollectionType(enum.Enum):
     TASKING = "TASKING"
 
 
-class IntegrationValue(enum.Enum):
-    ACCESS_APPROVAL_REQUIRED = "ACCESS_APPROVAL_REQUIRED"
-    SAMPLE_DATA_AVAILABLE = "SAMPLE_DATA_AVAILABLE"
-    MANUAL_REQUEST_REQUIRED = "MANUAL_REQUEST_REQUIRED"
-    FEASIBILITY_MAY_BE_REQUIRED = "FEASIBILITY_MAY_BE_REQUIRED"
-    QUOTATION_REQUIRED = "QUOTATION_REQUIRED"
-    PRICE_ESTIMATION_AVAILABLE = "PRICE_ESTIMATION_AVAILABLE"
-    SEARCH_AVAILABLE = "SEARCH_AVAILABLE"
-    THUMBNAIL_AVAILABLE = "THUMBNAIL_AVAILABLE"
-    QUICKLOOK_AVAILABLE = "QUICKLOOK_AVAILABLE"
+IntegrationValue: TypeAlias = Literal[
+    "ACCESS_APPROVAL_REQUIRED",
```

Comment:
Let's add the missing value

Reply:
I had replaced the value but now added it as an additional.